### PR TITLE
Added landing page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
Added a simple landing page for user to enter his name 
Name will be used to interaction purpose
Screenshot (277)

### CODE REVIEW

File: src/LandingPage/LandingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import "./LandingPage.css";
+import { useState } from "react";
+
+const LandingPage = () => {
+  const [name, setName] = useState("");
+  console.log(name);
```
Comment (ramalingaa) [inline src/LandingPage/LandingPage.jsx:6]: remove console here

File: src/LandingPage/LandingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        )}
+      </div>
+    </>
+  );
+};
+
+export { LandingPage };
+
+// onClick={() => {
+//     localStorage.setItem("name", name);
```
Comment (ramalingaa) [inline src/LandingPage/LandingPage.jsx]: If not needed remove commented code

**Review by ramalingaa — APPROVED** [review]
Everything looks clean, no changes required.

**Review by manish591 — APPROVED** [review]
Everything is looking good

**Review by vinc3nati — APPROVED** [review]

**Review by varnita619 — APPROVED** [review]
Code is clean and modular
Ready to merge

**Pull request merged**